tutorials/documentation_materials/classiq_101/classiq_concepts/execute/advanced_execution/advanced_execution_user_guide.ipynb

```diff
@@ -0,0 +1,834 @@
+{
```

Comment:
<p>please write here the exact notation: <code>estimate(H,qprog)</code> or something like this. Thus to connect the equation to the actual programming.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>The notation <code>estimate(H,qprog)</code>is not correct.</p>

Reply:
<p>The second argument is optional, it is required only in the parametric case.</p>